zfec/cmdline_zunfec.py

```diff
@@ -3,18 +3,24 @@
 # zfec -- a fast C implementation of Reed-Solomon erasure coding with
 # command-line, C, and Python interfaces
 
-import os, sys
+from __future__ import print_function
 
-import argparse
-import filefec
+import os, sys, argparse
+
+from zfec import filefec
 
 from zfec import __version__ as libversion
 __version__ = libversion
 
+try:
+    unicode = unicode
+except NameError:
+    unicode = str
```

Comment:
also, I'ld put such stuff into a _compat.py module, so it can be imported from there to anywhere it is needed.

same applies for:
```
bytes_type = bytes  # python 3
bytes_type = str  # python 2
```